packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -40,11 +40,12 @@ export type ProductOption = {
     monthlyAmount: number;
     monthlyFormatted: string;
   };
-  currencyCode: string;
-  currencySymbol: string;
+  currencyCode?: string;
+  currencySymbol?: string;
   extraLabel: string;
   appsId: PlusPriceTypeAppsId;
   duration: PlusPriceType;
+  durationLabel: 'month' | 'year';
```

Comment:
made sure its passed for paddle stuff?

Reply:
Yeah, set it to `month` as we're doing those wonky things there
https://github.com/dailydotdev/apps/blob/e376634a1beca66410d247c4251a8a7c9677cfc7/packages/shared/src/contexts/PaymentContext.tsx#L221